marrow/mailer/transport/smtp.py

```diff
@@ -68,10 +70,17 @@ def shutdown(self):
 
     def connect_to_server(self):
         if self.tls == 'ssl': # pragma: no cover
-            connection = SMTP_SSL(local_hostname=self.local_hostname, keyfile=self.keyfile,
-                                  certfile=self.certfile, timeout=self.timeout)
+            if PY35_OR_LATER:
+                connection = SMTP_SSL(local_hostname=self.local_hostname, keyfile=self.keyfile,
+                                      certfile=self.certfile, timeout=self.timeout)
+            else:
+                connection = SMTP_SSL(local_hostname=self.local_hostname, keyfile=self.keyfile,
+                                      certfile=self.certfile, timeout=self.timeout)
```

Comment:
Why is there a conditional branch here?  The invocations are identical.

Reply:
you're right there is an error in the second commit, sorry about that, the second commit was added because the travis continuous integration "tricked" me in thinking the first one was breaking python 2 and python < 3.5, but then looking in detail I saw that what seems broken is the continuous integration for some versions of Python. So you should really only pull the first commit. Would you prefer I close this pull request and submit another one?

Reply:
No need; an additional commit is no worries.  Squash merges are nice.  😉